x-pack/plugins/security_solution/public/rules/routes.tsx

```diff
@@ -18,33 +19,45 @@ const RulesSubRoutes = [
   {
     path: '/rules/id/:detailName/edit',
     main: EditRulePage,
+    exact: true,
   },
   {
     path: '/rules/id/:detailName',
     main: RuleDetailsPage,
+    exact: true,
   },
   {
     path: '/rules/create',
     main: CreateRulePage,
+    exact: true,
   },
   {
     path: '/rules',
-    exact: true,
     main: RulesPage,
+    exact: true,
   },
 ];
 
 export const RulesRoutes = () => {
   return (
-    <TrackApplicationView viewId={SecurityPageName.rules}>
-      <Switch>
-        {RulesSubRoutes.map((route, index) => (
-          <Route key={`rules-route-${route.path}`} path={route.path} exact={route?.exact ?? false}>
-            <route.main />
-          </Route>
-        ))}
-      </Switch>
-    </TrackApplicationView>
+    <Switch>
+      <Route path={RULES_PATH}>
```

Comment:
We already passing `RULES_PATH` path from the exported `routes`. These first _Switch_ and _Route_ here are not needed.